<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PIManagement</class>
 <widget class="QMainWindow" name="PIManagement">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1395</width>
    <height>940</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#centralwidget{
background:url(:/res/LW_12.jpg)
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>100</y>
      <width>1021</width>
      <height>811</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QFrame{
background:rgba(255,255,157,0.70);
border-radius:15px;
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>30</y>
       <width>351</width>
       <height>111</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel{
color:#00A388;
font-size:50px;
font-family:Microsoft JhengHei;
background:transparent;
}
</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:36pt; font-weight:600; font-style:italic; color:#5555ff;&quot;&gt;个 人 信 息&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>690</x>
       <y>650</y>
       <width>131</width>
       <height>131</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
background:#96CA2D;
border-radius:60px;
font-size:28px;
font-family:Microsoft JhengHei;
}
QPushButton:hover{
background:#168039;
border-radius:60px;
font-size:28px;
font-family:Microsoft JhengHei;
}
</string>
     </property>
     <property name="text">
      <string>修改</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>870</x>
       <y>650</y>
       <width>131</width>
       <height>131</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
background:#ff8c69;
border-radius:60px;
font-size:28px;
font-family:Microsoft JhengHei;
}
QPushButton:hover{
border-radius:60px;
background:#FF6F69;
font-size:28px;
font-family:Microsoft JhengHei;
}
</string>
     </property>
     <property name="text">
      <string>返回</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>140</y>
       <width>801</width>
       <height>341</height>
      </rect>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>60</y>
        <width>161</width>
        <height>91</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel{
color:white;
font-size:32px;
font-family:Microsoft JhengHei;
background:transparent;
}</string>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:24pt; color:#aa0000;&quot;&gt;用户名：&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_3">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>70</y>
        <width>451</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit{
background:transparent;
border:none;
color:#717072;
font-size:40px;
font-family:Microsoft JhengHei;
border-bottom:1px solid #717072;
}</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>220</y>
        <width>151</width>
        <height>101</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel{
color:white;
font-size:32px;
font-family:Microsoft JhengHei;
background:transparent;
}</string>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:24pt; color:#aa0000;&quot;&gt;密码：&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_4">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>230</y>
        <width>471</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit{
background:transparent;
border:none;
color:#717072;
font-size:40px;
font-family:Microsoft JhengHei;
border-bottom:1px solid #717072;
}</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>130</y>
       <width>951</width>
       <height>531</height>
      </rect>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>80</y>
        <width>231</width>
        <height>71</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel{
color:white;
font-size:32px;
font-family:Microsoft JhengHei;
background:transparent;
}</string>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:24pt; font-weight:600; color:#ff007f;&quot;&gt;Username：&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>210</y>
        <width>321</width>
        <height>141</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel{
color:white;
font-size:32px;
font-family:Microsoft JhengHei;
background:transparent;
}</string>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:24pt; font-weight:600; color:#ff007f;&quot;&gt;Password：&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>80</y>
        <width>471</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit{
background:transparent;
border:none;
color:#717072;
font-size:25px;
font-family:Microsoft JhengHei;
border-bottom:1px solid #717072;
}</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_2">
      <property name="geometry">
       <rect>
        <x>252</x>
        <y>260</y>
        <width>481</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit{
background:transparent;
border:none;
color:#717072;
font-size:25px;
font-family:Microsoft JhengHei;
border-bottom:1px solid #717072;
}</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_3">
      <property name="geometry">
       <rect>
        <x>630</x>
        <y>380</y>
        <width>131</width>
        <height>131</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
background:rgb(85, 85, 255);
border-radius:60px;
font-color:white;
font-size:28px;
font-family:Microsoft JhengHei;
}
QPushButton:hover{
background:rgb(85, 255, 255);
border-radius:60px;
font-size:28px;
font-family:Microsoft JhengHei;
}
</string>
      </property>
      <property name="text">
       <string>保存</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>150</y>
        <width>411</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel{
color:white;
font-size:32px;
font-family:Microsoft JhengHei;
background:transparent;
}</string>
      </property>
      <property name="text">
       <string>15位以内数字与字母的组合</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>320</y>
        <width>411</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel{
color:white;
font-size:32px;
font-family:Microsoft JhengHei;
background:transparent;
}</string>
      </property>
      <property name="text">
       <string>15位以内数字与字母的组合</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_4">
      <property name="geometry">
       <rect>
        <x>810</x>
        <y>380</y>
        <width>131</width>
        <height>131</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
background:rgb(255, 255, 0);
border-radius:60px;
font-color:yellow;
font-size:28px;
font-family:Microsoft JhengHei;
}
QPushButton:hover{
background:rgb(85, 255, 255);
border-radius:60px;
font-size:28px;
font-family:Microsoft JhengHei;
}
</string>
      </property>
      <property name="text">
       <string>取消</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
